packages/commonwealth/client/scripts/helpers/localStorage.ts

```diff
@@ -0,0 +1,39 @@
+const KEY_REFCODE = 'common-refcode';
+const REFCODE_EXPIRATION_DAYS = 7;
```

Comment:
we can use cookies if we want to handle expiry, they will auto delete this key once the set expiry date is passed.

Reply:
in this case I decided to stick with LS, lets try it out